matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/registration/RegistrationWizard.kt

```diff
@@ -110,4 +117,10 @@ interface RegistrationWizard {
      * called successfully.
      */
     val isRegistrationStarted: Boolean
+
+    /**
+     * True when login and password have been sent with success to the homeserver, i.e. [createAccount] has been
+     * called successfully.
+     */
+    val currentRegistrationSessionId: String?
```

Comment:
The doc is not correct (copy paste).

**But:**
I would prefer to avoid exposing the sessionId to the app.

The `registrationOther` implementation could add the `sessionId` parameter to the JsonDict before sending the request to the homeserver. WDYT?